allensdk/brain_observatory/receptive_field_analysis/receptive_field.py

```diff
@@ -36,56 +36,69 @@
 from .eventdetection import detect_events
 from statsmodels.sandbox.stats.multicomp import multipletests
 import numpy as np
-from .utilities import get_A, get_A_blur, get_shuffle_matrix, get_components, dict_generator
+from .utilities import get_A, get_A_blur, get_shuffle_matrix, get_components, \
+    dict_generator
 from .postprocessing import run_postprocessing
 import h5py
 
-def events_to_pvalues_no_fdr_correction(data, event_vector, A, number_of_shuffles=5000, response_detection_error_std_dev=.1, seed=1):
 
+def events_to_pvalues_no_fdr_correction(data, event_vector, A,
+                                        number_of_shuffles=5000,
+                                        response_detection_error_std_dev=.1,
+                                        seed=1):
     number_of_pixels = A.shape[0] // 2
 
     # Initializations:
     number_of_events = event_vector.sum()
     np.random.seed(seed)
 
-    shuffle_data = get_shuffle_matrix(data, event_vector, A, number_of_shuffles=number_of_shuffles, response_detection_error_std_dev=response_detection_error_std_dev)
+    shuffle_data = get_shuffle_matrix(
+        data, event_vector, A,
+        number_of_shuffles=number_of_shuffles,
+        response_detection_error_std_dev=response_detection_error_std_dev)
 
     # Build list of p-values:
-    response_triggered_stimulus_vector = A.dot(event_vector)/number_of_events
+    response_triggered_stimulus_vector = A.dot(event_vector) / number_of_events
     p_value_list = []
-    for pi in range(2*number_of_pixels):
-        curr_p_value = 1-(shuffle_data[pi, :] < response_triggered_stimulus_vector[pi]).sum()*1./number_of_shuffles
+    for pi in range(2 * number_of_pixels):
```

Comment:
I would just like to point out that we are exploring universes in which pi has different values...